### DISCUSSION TITLE
New Gameplay/Building Ideas for a more realistic and engaging Caesar 3.

### DISCUSSION THREAD

**Discussion by Purakane** [Keriew/augustus#446] [Ideas]
Hey Guys,
I've been playing and coming back to caesar 3 for 2 decades now I'm so happy that we have a community here that's been improving this incredible game after al these years. I also have a few suggestions for the next update if the development team are interested. 

Taverns 2x1 (rotatable) - increase happiness of citizens and serves wine. Because this is a place of business as well, having these near ports/ warehouses increase profit by a % or increase in total amount of goods sold annually.

Recruiting stations 1x1- A building which recruits conscripts and displays a pool of how many man are available to become soldiers/century guards. Having an unlimited supply of soldiers/guards always seemed a tad cartoonish.

brothel 2x1 - Increase recruitment rate for fighting men and moral boost for soldiers. Change the skin for the bath house lady.

Have the grand temple for Neptune to hold water and fountains that require it to be connected to a water supply. We all love water based buildings.

This one is a bit ambitious and I doubt it would work but have custom maps in which the bottom 1/5th of the map is divided by a black bar. Essentially the top 4/5 is your city and the bottom 1/5 is a battlefield for when caesar requests troops for invasions. Theses away battles can now be controlled in real time.

Please also add a hotkey for Undo.

**Comment by PaxBotanica** [comment]
I can see Taverns as 2x2 are now in game, as are recruiting/supply post. Would love to see Brothels as added entertainment and decrease unemployment and crime but perhaps with negative desirability to offset. 

Neptune is in now too. 

It think Public/Private Latrines and Grand Baths should definitely be in too if not in current development (mentioned elsewhere).

For desirability I would like to suggest two basic additions; 1) a possible alternative plaza options; going from caesar 4 a plain white marble paved roads 2) pink gardens plain paved area to matching existing gardens in the game. 

Further to this, I think it would be really cool to see roofed single slanted or dual slanted porticos that you could also add to roadway desirability. These were common around roman fora and major public causeways, and I think this might be manageable with existing graphics resources like the colonnade.

thanks for listening.
